source/iostreams.tex

```diff
@@ -10036,8 +10036,8 @@
 A sequence of elements that identify
 the location of a file within a filesystem.
 The elements are the
-\grammarterm{root-name\opt},
-\grammarterm{root-directory\opt},
+\grammarterm{\opt{root-name}},
+\grammarterm{\opt{root-directory}},
```

Comment:
Yes, I believe it should.  I see other places where grammarterm has problems as well, as can be seen in this grep:
```
$grep "grammarterm.*\<opt\>" *.tex
declarations.tex:\tcode{typename}\opt{} \grammarterm{nested-name-specifier}\opt{} \grammarterm{template-name}
declarations.tex:``\tcode{enum} \grammarterm{nested-name-specifier\opt} \grammarterm{identifier}''
declarations.tex:\grammarterm{attribute-specifier-seq\opt} \grammarterm{identifier}~(\ref{dcl.type.elab}), or
declarators.tex:is of the form \grammarterm{attribute-specifier-seq\opt}
declarators.tex:\grammarterm{attribute-specifier-seq\opt}
declarators.tex:\grammarterm{cv-qualifier-seq\opt} \grammarterm{ref-qualifier\opt}
declarators.tex:\grammarterm{cv-qualifier-seq}\opt \grammarterm{ref-qualifier}\opt
expressions.tex:parameter-type-list \cvqual{cv} \grammarterm{ref-qualifier\opt} returning \tcode{T}'', then
iostreams.tex:\grammarterm{root-name\opt},
iostreams.tex:\grammarterm{root-directory\opt}, 
```

Another editorial issue?

fwiw, I think the above should instead be coded as:
```
\grammarterm{root-name}\opt{},
\grammarterm{root-directory}\opt{},
```